run-profile-partitions

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+# Assume that assignments have been generated into ../zeth/build
+
+ASSIGNMENTS_DIR=../zeth/build
```

Comment:
If these scripts assume Zeth is used under the hood, shall we rename them to account for that? E.g. `run-profile-partitions` -> `run-zeth-profile-partitions`? (setting the snake case vs kebab case discussion aside here)